packages/Breadcrumbs/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
```

Comment:
We can add `shared-typography` to devDependencies, this may require a rebase.

Reply:
Shared-typography still needs to be added in order to receive proper version bumps from lerna.